src/frontend/views/modules/header/header.pug

```diff
@@ -6,7 +6,7 @@ div
       ul.list-inline.all-centered
         li.logo
           a(href='', ui-sref='home')
-            img.openpaas-logo(ng-src="{{ baseUrl + '/api/themes/' + domainId + '/logo'}}", fallback-src='assets/images/white-logo.svg', src='assets/images/white-logo.svg', alt="{{ 'OpenPaas Logo' | translate }}")
+            img.openpaas-logo(ng-src=`{{logoUrl}}`, src='assets/images/throbber-amber.svg', alt="{{ 'OpenPaas Logo' | translate }}")
```

Comment:
yes i agree, @oubchid can you replace it with white logo.

i may ask the same for avatars where "throbber" is not convenient (load letter instead of it)

Reply:
I will replace it